app/component/QuickSettingsPanel.js

```diff
@@ -163,6 +163,10 @@ class QuickSettingsPanel extends React.Component {
     const applicableQuickOptionSets = getApplicableQuickOptionSets(
       this.context,
     );
+    // isCarpool will be true if carpool was chosen as streetMode
+    const isCarpool = this.context.router.location.query.modes.startsWith(
```

Comment:
I would feel better if this didn't rely on a specific order, ie. it works with both `CARPOOL,WALK` and `WALK,CARPOOL`.

Reply:
Or even better, you could make it part of the configuration.

Reply:
I found this solution better because otherwise the second row would disappear (even when we don't want it to). For example if park+ride is selected as street mode but carpool is enabled as transport mode (in the second row), their order will be: park_ride, carpool. So the second row disappears if we click on carpool (in the second row) and we cannot disable it.

Reply:
And it isn't possible to add a flag to the config?